experimental/ExtendTeamsforM365/client/pages/categories.html

```diff
@@ -6,6 +6,15 @@
     <title>Northwind Product Categories</title>
     <link rel="stylesheet" href="/northwind.css" />
     <link rel="icon" href="data:;base64,="> <!-- Suppress favicon error -->
+    <style>               
+        #categoriesImgae{        
+        width: 4rem;
+        }   
+
+        #categoriesDescription{        
```

Comment:
Let's not add inline css in the html, instead add it to CSS file. (northwind-teams.css)

Reply:
Changed and shifted to northwind-teams.css